understanding/focus-appearance.html

```diff
@@ -53,389 +53,316 @@
                      </p>
                      
                      
                      <figure id="focus-outline-solid">
-                        <img src="img/focus-indicator-solid-outline.svg" alt="Three of 5 stars are selected with a solid-line focus indicator in the shape of a star outlining the third" />
+                        <img src="img/focus-indicator-solid-outline.svg" alt="五つの星のうち三つが選択され、3 番目の星のアウトラインとして星形の実線のフォーカスインジケータが表示されている。" />
                         
-                        <figcaption>Figure 2 Passes: a solid outline indicator surrounds the third of five stars.</figcaption>
+                        <figcaption>図 2 合格: 実線のアウトラインインジケータが五つの星のうち三つ目を囲んでいる。</figcaption>
                         
                      </figure>
                      
                      
                      <figure id="focus-border-solid">
-                        <img src="img/focus-indicator-solid-border.svg" alt="3 of 5 stars are selected with a solid-line rectangular focus indicator around the third" />
+                        <img src="img/focus-indicator-solid-border.svg" alt="五つの星の三つが選択され、3 番目の星の周りに実線の矩形のフォーカスインジケータが表示されている。" />
                         
-                        <figcaption>Figure 3 Passes: a solidly bound focus rectangle encloses the third of five stars.</figcaption>
+                        <figcaption>図 3 合格: 実線の境界フォーカス矩形が五つの星のうち三つ目を囲んでいる。</figcaption>
                         
                      </figure>
                      
                      
-                     <p>Offsetting indicators slightly from the focused component, as in the examples above,
-                        is not required to meet the minimum area requirement of the success criterion, but
-                        it can help make indicators more visible. In CSS, the <code>outline</code> and <code>outline-offset</code> properties are commonly used to achieve this.
+                     <p>上記の例のように、インジケータをフォーカスコンポーネントから少しずらすことは、達成基準の最小面積の要件を満たすために必要ではないが、インジケータをより見やすくするのに役立つ。CSS では、通例 <code>outline</code> 及び <code>outline-offset</code> プロパティがこれを達成するために使用される。
                      </p>
                      
                      
-                     <p>The smallest possible 2 CSS pixel thick indicator that is still a "perimeter" is a
-                        solid line that appears inside the component against the component's outer edge, for
-                        example by using a CSS <code>border</code> property. Indicators that are inset further within the component (not directly against
-                        the component's outer edge) need to be thicker than 2 CSS pixels to meet the minimum
-                        size requirement.
+                     <p>可能な限り小さい 2 CSS ピクセルの太さのインジケータで、「境界」であるものは、例えば CSS <code>border</code> プロパティを使うことで、コンポーネントの外辺に対してコンポーネントの内側に現れる実線である。(コンポーネントの外辺に直接ではなく) コンポーネント内にさらに挿入されるインジケータは、最小サイズの要件を満たすために 2 CSS ピクセルより太くする必要がある。
```

Comment:
perimeter なので「外周」としました。

また、「コンポーネント内にさらに挿入されるインジケータ」という表現がわかりにくそうな感じがしたので、少し手を入れてみました。

```suggestion
                     <p>可能な限り小さい 2 CSS ピクセルの太さのインジケータで、「外周」であるものは、例えば CSS <code>border</code> プロパティを使うことで、コンポーネントの外辺に対してコンポーネントの内側に現れる実線である。(コンポーネントの外辺に直接ではなく) コンポーネントのさらに内側に挿入されるインジケータは、最小サイズの要件を満たすために 2 CSS ピクセルより太くする必要がある。
```